src/main/java/org/mitre/synthea/engine/Generator.java

```diff
@@ -234,6 +231,12 @@ private void init() {
       System.out.println("Modules: " + String.join("\n       & ", moduleNames));
       System.out.println(String.format("       > [%d loaded]", moduleNames.size()));
     }
+
+    if (Boolean.parseBoolean(
+        Config.get("growtherrors.enable", "false"))) {
```

Comment:
`growtherrors.enable = false` could probably be simplified to `growth_errors = false`